code/modules/shuttle/shuttle_creation/shuttle_creator.dm

```diff
@@ -33,11 +34,17 @@ GLOBAL_LIST_EMPTY(custom_shuttle_machines)		//Machines that require updating (He
 	//During designation
 	var/overwritten_area = /area/space
 	var/list/loggedTurfs = list()
-	var/loggedOldArea
+	var/area/loggedOldArea
 	var/area/recorded_shuttle_area
 	var/datum/shuttle_creator_overlay_holder/overlay_holder
 	//After designation
 	var/linkedShuttleId
+	var/turf/recorded_origin //The last remembered location of our airlock
+	var/turf/exit //Record the exterior turf next to the airlock to prevent modification designation
+
+/obj/item/shuttle_creator/examine(mob/user)
+	. = ..()
+	. += linkedShuttleId ? "It is linked to the [recorded_shuttle_area.name]." : "It currently has [loggedTurfs.len/10] tons designated."
```

Comment:
\*

Reply:
Ah, I forgot to delete that after adding in stats section to the new GUI. thanks for pointing that out.